rdflib/plugins/sparql/sparql.py

```diff
@@ -3,7 +3,7 @@
 import collections
 import itertools
 import datetime
-from collections import Mapping, MutableMapping
+from collections.abc import Mapping, MutableMapping
```

Comment:
The removal was done in Python 3.9 . The project has Python 2 as a classifier. Perhaps trying collections.abc and falling back to collections for python 2 will be a more compatible fix.

Reply:
we generally put stuff to support multiple versions in https://github.com/RDFLib/rdflib/blob/master/rdflib/compat.py

Reply:
moved to #959. Closing this pull request because it is now a duplicate.

Had to start a new pull request after creating a proper fork.